pysmt/type_checker.py

```diff
@@ -61,7 +60,25 @@ def __init__(self, env=None):
         self.set_function(self.walk_bv_rotate, op.BV_ROL, op.BV_ROR)
         self.set_function(self.walk_bv_extend, op.BV_ZEXT, op.BV_SEXT)
         self.set_function(self.walk_bv_comp, op.BV_COMP)
-        self.set_function(self.walk_string_length, op.LENGTH)
+        
+        self.set_function(self.walk_identity_string, op.STRING_CONSTANT)
+        self.set_function(self.walk_str_length, op.STR_LENGTH)
```

Comment:
The defaults mappings are defined in ```walkers/generic.py``` and inherited here. Therefore, you do not need to re-state them here. The idea here is that you might be to group operators that have the same type-checking rules together (see for example ```walk_bv_to_bool```).